src/wallet/walletdb.h

```diff
@@ -114,6 +113,7 @@ class CKeyMetadata
         nVersion = CKeyMetadata::CURRENT_VERSION;
         nCreateTime = 0;
         hdKeypath.clear();
+        seedFp.SetNull();
```

Comment:
This commit lost its authorship details, but it's pretty trivial, so meh.

Reply:
For this one it merged in just fine, but was not correct for zcash. I did a `git cherry-pick ... -n`, and a `git commit -e`. Not sure what happened...